autoPyTorch/pipeline/components/training/trainer/RowCutMixTrainer.py

```diff
@@ -39,7 +39,8 @@ def data_preparation(self, X: np.ndarray, y: np.ndarray,
         # It is unlikely that the batch size is lower than the number of features, but
         # be safe
         size = min(X.shape[0], X.shape[1])
```

Comment:
This should also be changed to size=X.shape[1] right ?

Reply:
True, I missed that